src/lib/Behat/PageElement/Tables/SubItemsTable.php

```diff
@@ -73,20 +74,22 @@ public function clickEditButton(string $listItemName): void
      *
      * @return bool
      */
-    public function isElementInTable(string $name, ?string $contentType = null): bool
+    public function getElementPositionInTable(string $name, ?string $contentType = null): int
     {
         if ($this->context->isElementVisible($this->fields['noItems'])) {
-            return false;
-        } else {
-            $tableHash = $this->getTableHash();
-            foreach ($tableHash as $row) {
-                if (($row['Name'] === $name) && (($row['Content type'] === $contentType) || ($contentType === null))) {
-                    return true;
+            return 0;
+        }
+        if (strpos($this->context->findElement($this->fields['list'])->getText(), $name) !== false) {
```

Comment:
If I understand correctly: this if checks if our text is present in the whole table, to speed up things? This requires a comment 😛 

Or introduce some variables(also: return early if possible to reduce nesting), for example:
```
$isTextPresentInTable = strpos($this->context->findElement($this->fields['list'])->getText(), $name) !=== false;
if(!isTextPresentInTable) {
return 0;
}
...
$actualContentType = $this->context->findElement(sprintf($this->fields['listElementType'], $matchingListElementsPositions[$i]))->getText();
...
```